app/src/androidTest/java/com/tari/android/wallet/FFICommsConfigTests.kt

```diff
@@ -82,7 +82,8 @@ class FFICommsConfigTests {
             DB_NAME,
             walletDir,
             Constants.Wallet.discoveryTimeoutSec,
-            Constants.Wallet.storeAndForwardMessageDurationSec
+            Constants.Wallet.storeAndForwardMessageDurationSec,
+            "weatherwax"
```

Comment:
We keep network names in Network enum
```suggestion
            "Network.WEATHERWAX.uriComponent"
```